### PR TITLE
DHFPROD-8431: Discard All Changes in Draft

### PR DESCRIPTION
**Pull request by AkshayBajajML**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- [x] Added Tests  

- ##### Reviewer:

- [x] Reviewed Tests
- [x] Added to Release Wiki

### CODE REVIEW

**Comment by AkshayBajajML** [comment]
@jbelonoj Reviewed and approved UI changes.
Screencast: https://drive.google.com/file/d/1LkF1M7J7pgcBukqmKSkuXDBQJFKX4WY6/view?usp=sharing

**Comment by AkshayBajajML** [comment]
Updated 5.7 release notes as well.

File: marklogic-data-hub-central/ui/src/components/modeling/graph-view/graph-view.tsx [hunk 1]
```diff
@@ -139,6 +149,22 @@ const GraphView: React.FC<Props> = (props) => {
     </span>
   </HCButton>;
 
+  const revertButton = <HCButton
+    className={props.canWriteEntityModel ? (!modelingOptions.isModified ? styles.disabledPointerEvents : "") : styles.disabledPointerEvents}
+    disabled={props.canWriteEntityModel ? !modelingOptions.isModified : true}
+    aria-label="revertChangesGraphView"
```
Comment (aniamy) [inline marklogic-data-hub-central/ui/src/components/modeling/graph-view/graph-view.tsx:155]: I noticed publish aria label pattern are different. Should we use the same pattern for aria labels? I don't know if the standard would be camel case or the snake case.

File: marklogic-data-hub-central/ui/src/pages/Modeling.tsx [hunk 2]
```diff
@@ -297,6 +317,24 @@ const Modeling: React.FC = () => {
   </HCButton>
   </span>;
 
+  const revertButton = <span className={styles.publishButtonParent}><HCButton
+    className={canWriteEntityModel ? (!modelingOptions.isModified ? styles.disabledPointerEvents : "") : styles.disabledPointerEvents}
+    disabled={canWriteEntityModel ? !modelingOptions.isModified : true}
+    aria-label="revertChangesTableView"
```
Comment (aniamy) [inline marklogic-data-hub-central/ui/src/pages/Modeling.tsx:323]: Same question about aria label pattern here.

File: marklogic-data-hub-central/ui/src/pages/Modeling.tsx [hunk 3]
```diff
@@ -159,6 +161,22 @@ const Modeling: React.FC = () => {
...
+    try {
+      let response = await clearDraftModels();
+      if (response["status"] === 200) {
+        await setEntityTypesFromServer();
+      }
+    } catch (error) {
+      handleError(error);
+    } finally {
+      clearEntityModified();
+      // toggleRelationshipModal(false);
```
Comment (aniamy) [inline marklogic-data-hub-central/ui/src/pages/Modeling.tsx:174]: Delete this commented line.

File: marklogic-data-hub-central/ui/src/components/modeling/graph-view/graph-view.tsx (hunk 1 above)
Comment (AkshayBajajML) [reply marklogic-data-hub-central/ui/src/components/modeling/graph-view/graph-view.tsx:155]: @aniamy I can see snake case in this file; I will change and keep it consistent

File: marklogic-data-hub-central/ui/src/pages/Modeling.tsx (hunk 2 above)
Comment (AkshayBajajML) [reply marklogic-data-hub-central/ui/src/pages/Modeling.tsx:323]: Same as above

**Review by aniamy — APPROVED** [review]

**Review by ryanjdew — DISMISSED** [review]

**Review by yubiselmarklogic — APPROVED** [review]

**Comment by AkshayBajajML** [comment]
Refactored the tests and now all pass except 1 expected intermittent one.

**Review by briantang — APPROVED** [review]

**Review by ryanjdew — APPROVED** [review]

**Pull request merged**